### DISCUSSION TITLE
Unsure how to Debug this error. an error is coming up from the createClient(supabaseUrl,supabaseAnonKey); function. Error:  Uncaught TypeError: ue.PostgrestClient is not a constructor

### DISCUSSION THREAD

**Discussion by KTayy** [supabase/supabase#26673] [Questions] [answered]
JS: 
```
import { createClient } from 'https://esm.sh/@supabase/supabase-js@2'
export const supabase = createClient(supabaseUrl,supabaseAnonKey);

```

HTML:
 ```

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>

    <!-- <script src="https://cdn.jsdelivr.net/npm/@supabase/supabase-js@2"></script> -->
    <!-- <script src="https://unpkg.com/@supabase/supabase-js@2"></script> -->

    <script src="https://cdn.jsdelivr.net/npm/@supabase/supabase-js@2"></script>
    <script type='module' src="../static/js/db/dbClient.js"></script>    
    <script type='module' src="../static/js/db/dbEvents.js"></script>
    <script type='module' src="../static/js/db/dbLoginHandler.js"></script>
    

</head>
<body>
    <form id="login-form">
        <input type="email" id="email" placeholder="Email">
        <input type="password" id="password" placeholder="Password">
        <button type="submit">Login</button>
    </form>
    <button id="signOutButton">Sign Out</button>
</body>
</html>
```
Full error message: 
```
Uncaught TypeError: ue.PostgrestClient is not a constructor
    j SupabaseClient.ts:106
    Oe index.ts:42
    <anonymous> dbClient.js:9
3 SupabaseClient.ts:106:16
    j SupabaseClient.ts:106
    Oe index.ts:42
    <anonymous> dbClient.js:9
```

the odd part is I have not touched this file in like 5 months, not sure how the error occurred

**Comment by GaryAustin1** [comment]
There was just a release of supabase-js that might be the cause: https://github.com/supabase/supabase-js/pull/1139
If you just pull the latest release all the time.

**Comment by teh0** [comment]
I have pulled the lastest version like this :
`import {createClient} from "https://esm.sh/@supabase/supabase-js@2.43.3";`

Still not working for me

**Reply by GaryAustin1** [reply]
The GUESS is that version is what is hurting you.  Go back to an older version.

**Reply by teh0** [reply]
Oh indeed ! It is working for me with version 2.43.2 : 
import {createClient} from "https://esm.sh/@supabase/supabase-js@2.43.2";

**Reply by nrose01** [reply]
Thank you for posting this! I have the same issue and I have been away from my project for a couple of weeks and could not figure out what the hell was going on.

**Reply by isaacR23** [reply, marked as answer]
1. Change import { createClient } from "https://esm.sh/@supabase/supabase-js@2"; -> import { createClient } from "https://esm.sh/@supabase/supabase-js@2.43.2";
2. supabase stop
3. supabase start
Done

**Reply by KTayy** [reply]
Thanks Guys! I had a feeling it was something to do with an update.

**Comment by soedirgo** [comment]
This should also be resolved as of v2.43.4
